cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml

```diff
@@ -350,4 +326,3 @@ spec:
       for: 1m
       labels:
         severity: critical
-
```

Comment:
keep the new line? or is it auto-generated?

Reply:
that should be a copy-paste mistake. added the last newline

Reply:
added the newline...